### ISSUE TITLE
The discord administrator arbitrarily blocks users?

### ISSUE THREAD

**Issue by krishuang0610** [madMAx43v3r/mmx-node#174]
The discord administrator arbitrarily blocks users?

**Comment by mwpastore** [comment]
Your Discord account has been compromised and is being used to spam all the channels in the madMAx server. Please take immediate action to secure your account. 

![capture01](https://user-images.githubusercontent.com/8540936/220478280-5127012d-dd15-4745-bc57-f57a725d340b.png)

**Comment by madMAx43v3r** [comment]
yeah sorry, we thought you're a bot or an old hacked account. let me know when you've changed your password and I unban you.

**Comment by krishuang0610** [comment]
My God. I'm sorry, I didn't find out that my account was abnormal in time, I have changed the account password now, thank you for your help and reminder

**Comment by madMAx43v3r** [comment]
Ok, I removed the ban now.
